mlx_graphs/nn/conv/simple_conv.py

```diff
@@ -0,0 +1,82 @@
+from typing import Literal, Optional, get_args
+
+import mlx.core as mx
+
+from mlx_graphs.nn.message_passing import MessagePassing
+from mlx_graphs.utils import ScatterAggregations, add_self_loops
+
+CombineRootFunctions = Literal["sum", "cat", "self_loop"]
+
+
+class SimpleConv(MessagePassing):
+    """A simple non-trainable message passing layer.
+
+    Inspired by the SimpleConv PyG layer `here \
+    <https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.SimpleConv.html>`_.
+
+    Args:
+        aggr: Aggregation strategy used to aggregate messages,
+            e.g. "add", "mean", "max". Default: ``add``
+        combine_root_func: Strategy used to combine the features from the root nodes
+        (values: "sum", "cat", "self_loop" or `None`). Default: `None`
```

Comment:
It would be great to explain how each function operates

Reply:
Fixed!